README.rst

```diff
@@ -449,6 +449,74 @@ Tests execution
 2) Assign the PYTHONPATH environment variable executing "export PYTHONPATH=../.."
 3) Run lettuce_tools with appropriate params (see available ones with the -h option)
 
+Skuld Testbed
+-------------
+
+A set of scripts to create a secure environment for testing is provided. The
+environment is created in a virtual machine using the FIWARE Lab infrastructure.
+The environment variables OS_AUTH_URL, OS_REGION_NAME, OS_USERNAME, OS_PASSWORD
+and OS_TENANT_NAME must be filled with the data of a valid FIWARE Lab account.
+
+The installation of the testbed is fully automatized and consists on an OpenStack
+instance where the original keystone server was replaced with the KeyRock server.
+This is work in progress; the current version only installs Glance, Nova and Neutron,
+but skuld also purges Swift, Cinder and Blueprint resources.
+
+To install, it is only necessary to configure the same virtualenv than for
+running skuld, setting PYTHONPATH with the path of the project and invoking the
+script *tests/install_testbed/launch_vm.py*.
+
+The script uses a floating IP and creates both a keypair (it saves the SSH private key as
+~/.ssh/testbedskuld_key) and a security group. The security group opens
+the port 22 (SSH) and allows the ICMP traffic. It is also possible to connect
+to any port from a VM using the same security group.
+
+The names of the keypair, the security group and other parameters as the preferable
+floating IP are configurable in the *settings.py* file, but most of the users may
+ignore this file securely. However, to use the script in a different platform
+than FIWARE Lab, probably is necessary to change the parameter about the
+public shared network name.
+
+Using the testbed
+*****************
+
+The *launch_vm.py* ends in a few seconds, showing the floating IP. Although it is
+already possible to connect to the ubuntu account of the server (using the
+ SSH key at *~/.ssh/testbedskuld_key*), the installation is still running
+inside the VM and needs a few minutes to complete. Usually the installation
+process last between 10 and 20 minutes. The job is finish after the file
+config_vars is copied into the */home/ubuntu* folder inside the virtual machine.
+
+When the installation is finished, the credential may be loaded with *. ~/config_vars*.
+The command *nova list* shows a testing VM that has been created during the installation
+inside the testbed (that is, it is a virtual machine running inside the testbed
+virtual machine). The floating IP 192.168.58.201 is associated to this
```

Comment:
So, I recomend to write this comment in the README file in order to clarify it. Take into account that someone that comes here and see this IP cannot associate to the pool, so I describe that we create an IP pool with IP ... and the first one (...) of this pool is associated to the router.